index.d.ts

```diff
@@ -225,17 +225,17 @@ export interface Chalk {
 	readonly inverse: this;
 
 	/**
-	 * Modifier
+	 * Modifier: Prints the text, but makes it invisible.
 	 */
 	readonly hidden: this;
 
 	/**
-	 * Modifier
+	 * Modifier: https://en.wikipedia.org/wiki/Strikethrough
```

Comment:
Sorry, I was unclear. I didn't mean to use this literally, but rather to find a good description from the Wikipedia page.

> Puts a horizontal line through the center of the text.

It should also contain the `(Not widely supported)` part from the readme.